packages/angular_devkit/architect/src/api.ts

```diff
@@ -250,8 +250,16 @@ export interface BuilderContext {
 /**
  * An accepted return value from a builder. Can be either an Observable, a Promise or a vector.
  */
-export type BuilderOutputLike = Observable<BuilderOutput> | Promise<BuilderOutput> | BuilderOutput;
+export type BuilderOutputLike = SubscribableOrPromise<BuilderOutput> | BuilderOutput;
 
+// tslint:disable-next-line:no-any
+export function isBuilderOutput(obj: any): obj is BuilderOutput {
```

Comment:
Wouldn't this be better off with `unknown`?

Reply:
Initially went with that approach but the code become several times larger with the additional required checks to test the properties.